src/app/components/client/toolbar/AppPicker.tsx

```diff
@@ -220,33 +220,32 @@ type AppPickerTriggerProps = {
   MenuTriggerProps &
   AriaMenuTriggerProps;
 
-// TODO: Add unit test when changing this code:
-/* c8 ignore start */
 function AppPickerTrigger(props: AppPickerTriggerProps) {
   const l10n = useL10n();
-  const state = useMenuTriggerState({});
+  const state = useMenuTriggerState({
+    onOpenChange: (isOpen) => {
+      isOpen &&
+        gaEvent({
+          category: "bento",
+          action: state.isOpen ? "bento-opened" : "bento-closed",
+          label: props.referringHost,
+        });
+    },
+  });
   const ref = useRef<HTMLButtonElement | null>(null);
```

Comment:
I think you'll want to remove the `isOpen &&` - now it doesn't send an event when the menu gets closed :sweat_smile:

Reply:
Oh, and `isOpen` is passed as an argument to `onOpenChange` - I think you'll want to use that, instead of `state.isOpen`, because the callback might get called before the `state` object updates.